src/Filter/StringFilter.php

```diff
@@ -115,4 +121,19 @@ private function getOperator(int $type): string
     {
         return self::CHOICES[$type] ?? self::CHOICES[StringOperatorType::TYPE_CONTAINS];
     }
+
+    private function trim(string $string): string
+    {
+        $trim_mode = $this->getOption('trim', self::TRIM_BOTH);
```

Comment:
Shouldn't trim be declared in `getDefaultOptions` ?

Reply:
fixed